app/models.py

```diff
@@ -1,39 +1,38 @@
-from poplib import POP3_SSL_PORT
-from turtle import update
-from unittest import result
-from flask import session
 from werkzeug.security import generate_password_hash, check_password_hash
 
 from datetime import datetime
 
-from app import db
+from flask_login import UserMixin
+
+from app import db, login
+
 
 class BaseModel(db.Model):
+    __abstract__ = True
+
     id = db.Column(db.Integer, primary_key=True)
 
-    def find_by_id(self):
-        pass
+    @classmethod
+    def find_by_id(cls, _id):
+        return cls.query.filter_by(id=_id).first()
 
-    def find_by_name(self):
-        pass
-    
-    # TODO - Define __schema__ class
-    # TODO - Create base model and inheritt to all models
-    # id is a common field for every new model. You don't
-    # have to create it every time.
+    def save_to_db(self):
```

Comment:
always create base models with save and delete actions this will save you lines of code